src/main.rs

```diff
@@ -89,20 +93,33 @@ impl Service for Contributors {
                     f.read_to_string(&mut source).unwrap();
 
                     use contributors::schema::commits::dsl::*;
+                    use contributors::models::Commit;
 
                     let connection = contributors::establish_connection();
 
-                    let mut names: Vec<String> = commits.select(author_name).distinct().load(&connection).unwrap();
+                    // It's possible to do this in Postgres with
+                    // SELECT author_name, COUNT(author_name) as commit_count FROM commits GROUP BY author_name ORDER BY commit_count DESC;
+                    // but it doesn't look like Diesel supports that
```

Comment:
Will have a look after dinner since I had to fiddle a bit to get this compiling, since hyper's tokio branch was removed. This is a commit that seems to work:

```toml
[dependencies.hyper]
git = "https://github.com/hyperium/hyper"
rev = "6e4739f918dfb53dff881717a96cfd6b73d06ba8"
```